projects/mno-frontend-lib/src/lib/_models/cache/cache-object.ts

```diff
@@ -0,0 +1,25 @@
+export interface CacheConfig {
```

Comment:
@cesar-tonnoir it doesn't have to be in-place of, it would be 1 line to add `localStorage.setItem` to the internal setter of the `CacheService` if you want local persistence. The `@Cache()` decorator is just an interface to cache the result of the observable function it's decorating. 

Think about having to write localStorage.setItem for each observable you don't want multiple requests to fire for when multiple parts of the applications are subscribing to. 

```js
myFetchFn() {
   if (localStorage.getItem('blah') {
      return of(JSON.parse(localStorage.getItem('blah')))
   } else  {
        this.http.get().tap(res => {
            localStorage.setItem('blah', JSON.stringify(res))
   ...

// vs

@Cache('blah')
myFetchFn()
   return this.http.get()
```